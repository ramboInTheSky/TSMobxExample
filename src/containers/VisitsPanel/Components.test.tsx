import React from 'react'
import { shallow } from 'enzyme'

import {
	Wrapper,
	Entry,
	AccessContainer,
	DatesContainer,
	Access,
	Status
} from './Components'

describe('<Wrapper />', () => {
	it('renders correctly', () => {
		expect(shallow(<Wrapper />)).toMatchSnapshot()
	})
})

describe('<Entry />', () => {
	it('renders correctly', () => {
		expect(shallow(<Entry />)).toMatchSnapshot()
	})
})

describe('<AccessContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<AccessContainer />)).toMatchSnapshot()
	})
})

describe('<DatesContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<DatesContainer />)).toMatchSnapshot()
	})
})

describe('<Access />', () => {
	it('renders correctly', () => {
		expect(shallow(<Access />)).toMatchSnapshot()
	})
})

describe('<Status />', () => {
	it('renders correctly', () => {
		expect(shallow(<Status />)).toMatchSnapshot()
	})
})
