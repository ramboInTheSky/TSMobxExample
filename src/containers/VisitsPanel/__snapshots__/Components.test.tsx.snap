// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Access /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<AccessContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<DatesContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Entry /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Status /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Wrapper /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
